tools/wptrunner/wptrunner/executors/protocol.py

```diff
@@ -28,6 +28,18 @@ def __init__(self, executor, browser):
             assert not hasattr(self, name)
             setattr(self, name, cls(self))
 
+    def update(self, target, source):
+        if (type(target) is not dict
+                or type(source) is not dict):
```

Comment:
We don't use this extra-intent-to-avoid-if-and-condition-lining-up style anywhere else in this code.

Reply:
updated, thanks!